subiquity/server/server.py

```diff
@@ -112,14 +113,15 @@ async def mark_configured_POST(self, endpoint_names: List[str]) -> None:
                 controller.configured()
 
     async def client_variant_POST(self, variant: str) -> None:
```

Comment:
So I have a PR that removes this API... we can just make it only supported in the system_setup code I guess. It also feels a bit wrong to me to have the client decide whether this is setup or reconfiguration but well.

Reply:
Agreed, the architecture the server will decide which variant it is executing.
Lets merge this, then adapt the PR to the new API  both subiquity and system_setup afterwards.